src/main/java/org/reactome/release/Main.java

```diff
@@ -0,0 +1,35 @@
+package org.reactome.release;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.List;
+import org.reactome.release.resourcechecker.FTPFileResourceChecker;
+import org.reactome.release.resourcechecker.FileResourceChecker.ByteUnit;
+import org.reactome.release.resourcechecker.ResourceChecker;
+import org.reactome.release.resourcechecker.ResourceCheckerFactory;
+
+public class Main {
+	public static void main(String[] args) throws Exception {
+		System.out.println("ResourceExists\tFileSize");
+		for (Resource resourceToCheck: getResourcesToCheck("External_Resources_Single_Test_Resource.json")) {
+			ResourceChecker resourceChecker = ResourceCheckerFactory.getInstance(resourceToCheck);
+//			System.out.println("Host: " + resourceToCheck.getResourceURL().getHost());
+//			System.out.println(resourceToCheck.getResourceURL() + " exists? " + resourceChecker.resourceExists());
+			System.out.println(resourceChecker instanceof FTPFileResourceChecker ? ((FTPFileResourceChecker) resourceChecker).getFileSizeAs(
```

Comment:
Are these commented out lines eventually going to be used or can they be removed?

Reply:
The `Main` class is a work in progress, so all print statements will probably be removed.  It's being used for running the application just for testing right now.